app/src/shared/components/FieldList/FieldItem/index.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react'
+
+import TextInput from '$shared/components/TextInput'
+import Handle from '../Handle'
+import styles from './fieldItem.pcss'
+
+type Props = {
+    name: string,
+}
+
+const FieldItem = ({ name }: Props) => (
+    <div className={styles.root}>
+        <div className={styles.inner}>
+            <Handle className={styles.handle} />
+            <TextInput label="" defaultValue={name} />
```

Comment:
How does one listen for changes on the `TextInput`? I'm thinking perhaps `TextInput` should be `{children}`?

Reply:
Just an example, yes.